pkg/operation/botanist/controlplane/etcd/monitoring.go

```diff
@@ -374,10 +348,6 @@ func (e *etcd) AlertingRules() (map[string]string, error) {
 // CentralMonitoringConfiguration returns scape configuration for the central Prometheus.
 func CentralMonitoringConfiguration() (component.CentralMonitoringConfig, error) {
```

Comment:
Can you remove the whole function if it is not needed now anymore?

/invite @istvanballok